igor-web/src/main/groovy/com/netflix/spinnaker/igor/wercker/WerckerService.groovy

```diff
@@ -171,16 +172,18 @@ class WerckerService implements BuildOperations {
 
                 //return the integer build number for this run id
                 return runIdBuildNumbers.get(run.id)
-            } catch (RetrofitError e) {
-                def body = e.getResponse().getBody()
-                String wkrMsg
+            }  catch (SpinnakerServerException e) {
+              String wkrMsg
+              if(e instanceof  SpinnakerHttpException){
+                def body = e.getResponseBody()
                 if (body instanceof TypedByteArray) {
```

Comment:
This is a little tricky...SpinnakerHttpException doesn't provide access to the raw response body.  getResponseBody returns a map...so, I think we're looking at a change in behavior here.

That said, given
```
kv("errMsg", wkrMsg)
```
below, maybe it's cleaner to ditch all this here (including the e instanceof SpinnakerHttpException check) and do
```
kv("errMsg", e.getMessage())
```